corona_stats/packages/corona_stats/corona_stats.js

```diff
@@ -88,57 +162,93 @@ const fail = (err, msg) => {
   console.log(err);
   return {
     response_type: 'in_channel',
-    text: msg || "Couldn't get stats."
+    text: msg || "Couldn't get stats.",
   };
 };
 
 const success = (header, fields, footer) => {
   const response = {
     response_type: 'in_channel',
-    blocks: [
-      {
-        type: 'section',
-        text: {
-          type: 'mrkdwn',
-          text: `*${header}*`
-        }
+    blocks: [{
+      type: 'section',
+      text: {
+        type: 'mrkdwn',
+        text: `*${header}*`,
       },
-      {
-        type: 'section',
-        fields: []
-      }
-    ]
+    },
+    {
+      type: 'section',
+      fields: [],
+    },
+    ],
   };
   for (const property in fields) {
-    response.blocks[1].fields.push(
-      {
-        type: 'mrkdwn',
-        text: `${property}:   *${(fields[property] || 0)}*`
-      }
-    );
+    response.blocks[1].fields.push({
+      type: 'mrkdwn',
+      text: `${property}:   *${(fields[property] || 0)}*`,
+    });
   }
   if (footer) {
     response.blocks.push({
       type: 'context',
-      elements: [
-        {
-          type: 'mrkdwn',
-          text: footer
-        }
-      ]
-    })
+      elements: [{
+        type: 'mrkdwn',
+        text: footer,
+      }],
+    });
   }
   return response;
 };
 
+
+const getDetails = (name, html, domName) => {
+  const fields = {};
+  try {
+    tableParser(html);
+    const stats = html(`#${domName}`).parsetable(false, false, true);
+    if (stats.length === 0) {
+      return;
+    }
+    const recordIndex = stats[0].indexOf(name);
+    if (recordIndex > 0) {
+      fields['Total Cases'] = stats[1][recordIndex];
+      fields['New Cases'] = stats[2][recordIndex];
+      fields['Total Fatalities'] = stats[3][recordIndex];
+      fields['Total Recovered'] = stats[5][recordIndex];
+      fields['New Fatalities'] = stats[4][recordIndex];
+      fields['Active Cases'] = stats[6][recordIndex];
+      fields['Critical Cases'] = stats[7][recordIndex];
+    }
+  } catch (e) {
+    fail(e.message);
+  }
+  return fields;
+};
+
+async function getData(api, url) {
+  let response;
+  try {
+    response = await api.get(url);
+    if (response.status !== 200) {
+      return;
+    }
+  } catch (err) {
+    return fail(err.message);
+  }
+  if (!response) return;
+  return cheerio.load(response.data);
+}
+
+
 /**
  * @description Live stats for the epidemic, worldwide or in a specific country
  * @param {ParamsType} params list of command parameters
  * @param {?string} commandText text message
  * @param {!object} [secrets = {}] list of secrets
  * @return {Promise<SlackBodyType>} Response body
  */
-async function _command(params, commandText, secrets = {}) {
+
+async function _command(params) {
   const axios = require('axios');
```

Comment:
you can lift all the require and package installations to global scope so they are done once when the code is loaded - this would simplify what's in this `_command` function.